test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -47,7 +47,7 @@ var _ = g.Describe("ScyllaCluster replace", func() {
 		framework.By("Replacing a non-seed node")
 		pod, err := f.KubeClient().CoreV1().Pods(f.Namespace()).Get(
 			ctx,
-			getFirstNonSeedNodeName(sc),
+			getNodeName(sc, podIdx),
```

Comment:
Can we just replace those nodes in sequence instead of running the whole setup again?